kie-wb-common-screens/kie-wb-common-data-modeller/kie-wb-common-data-modeller-backend/src/main/java/org/kie/workbench/common/screens/datamodeller/backend/server/DataModelerServiceImpl.java

```diff
@@ -569,6 +566,19 @@ public GenerationResult saveSource(final String source,
                 targetName = newFileName + ".java";
                 nameChanged = true;
             }
+            
+            /* Modify data object with updated fileName and packageName
+             * or by resolving from @path before resolving source to handle
+             * inconsistencies in source code and data object
+             * */
+            if  (dataObject  != null) {
```

Comment:
```suggestion
            if (dataObject != null) {
```
Since we are keeping this `if` block, can you please remove these extra spaces?